bin/reformat-files.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Find all JS and JSX files in the project that have the @format tag
+ * and reformat them with Prettier. Useful when upgrading Prettier to
+ * a newer version.
+ * @format
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const glob = require( 'glob' );
+const ignore = require( 'ignore' );
+const docblock = require( 'jest-docblock' );
+const prettier = require( 'prettier' );
+
+// Load ignore file
+const ignoreFile = fs.readFileSync( '.eslintignore', 'utf-8' );
+const ig = ignore().add( ignoreFile );
+
+// List .js and .jsx files in the current directory
+const files = glob.sync( '**/*.{js,jsx}', {
+	ignore: 'node_modules/**', // ignore node_modules by default
+	nodir: true,
+} );
+
+files.forEach( file => {
+	// skip ignored files
+	if ( ig.ignores( file ) ) {
+		return;
+	}
+
+	const text = fs.readFileSync( file, 'utf8' );
```

Comment:
cool! I didn't know we could just leave this string here - I've been using `{ encoding: 'utf8' }`

Reply:
The credit goes to ESLint authors, as I copied most of this code from there :) They also use the `glob` and `ignore` libraries to list the right files to lint when running `eslint .`